Dockerfile

```
@@ -0,0 +1,18 @@
+FROM rust:1.56 AS builder
+
+WORKDIR /app
+
+COPY . ./
```

Can this be:

```suggestion
COPY . /app
```

Seems slightly clearer to me.